common/client/poller_test.go

```diff
@@ -89,7 +90,7 @@ func Test_Poller(t *testing.T) {
 			}
 			return true
 		}
-		require.Eventually(t, logsSeen, time.Second, time.Millisecond)
+		require.Eventually(t, logsSeen, tests.WaitTimeout(t), 100*time.Millisecond)
```

Comment:
This will change the timeout locally to 30s from 1s.  If we do this everywhere the local experience will be worse.  Instead of using a default in tests.WaitTimeout, can we have WaitTimeout take a default and use 1 here so local failures can be faster?

Reply:
I'm not opposed to using a shorter effective value, but I think it is important to derive it from the configured timeout one way or another so that there are not extra individual knobs to tune in each test.